client/src/locales/gb.json

```
@@ -445,73 +445,623 @@
+    "validationFailed": "Validation failed",
+    "noFileSelected": "No file selected",
+    "fallbackPage": "Import a file to upload a list of servers in bulk"
+  },
+  "publicLink": "Public link",
+  "maskedPageSpeedKeyPlaceholder": "*************************************",
+  "pageSpeedApiKeyFieldTitle": "Google PageSpeed API key",
+  "pageSpeedApiKeyFieldLabel": "PageSpeed API key",
+  "pageSpeedApiKeyFieldDescription": "Enter your Google PageSpeed API key to enable pagespeed monitoring. Click Reset to update the key.",
+  "pageSpeedApiKeyFieldResetLabel": "API key is set. Click Reset to change it.",
+  "reset": "Reset",
+  "pleaseCheckFormErrors": "Please check the form for errors.",
```

Biome (1.9.4)

[error] The key pleaseCheckFormErrors was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)